web/src/styles/components/buttons.scss

```diff
@@ -249,3 +249,10 @@
   padding: 0;
   text-decoration: underline !important;
 }
+
+.usa-icon-button-margin {
```

Comment:
This is used to control the spacing and padding when a launch icon is used inside a button